sp/src/game/server/ez2/npc_wilson.cpp

```diff
@@ -368,6 +380,42 @@ void CNPC_Wilson::Activate( void )
 			m_pMotionController->Enable();
 		}
 	}
+
+	RefreshCameraTargets();
+}
+
+//-----------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+bool CNPC_Wilson::KeyValue( const char *szKeyName, const char *szValue )
+{
+	bool bResult = BaseClass::KeyValue( szKeyName, szValue );
+
+	if( !bResult )
+	{
+		// Temporary until we can change saves and rename m_bOmniscient
+		if (FStrEq( szKeyName, "Omnipresent" ))
```

Comment:
I don't have a problem with adding "Omnipresent" as an alias but I don't like removing "Omniscient". I know it's probably never been used before, probably, but it's still a breaking change. I don't think the semantics of it are all that important - I knew what you meant when you said "Omniscient". It's close enough.